src/content/app/entity-viewer/variant-view/variant-image/alternative-allele/AlternativeAllele.tsx

```diff
@@ -0,0 +1,194 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import SequenceLetterBlock from '../sequence-letter-block/SequenceLetterBlock';
+
+import variantGroups from 'src/content/app/genome-browser/constants/variantGroups';
+
+import styles from './AlternativeAllele.module.css';
+
+type Props = {
+  regionSliceStart: number;
+  variantStart: number; // accounts for anchor base in appropriate variant types
+  variantLength: number; // length of reference allele; accounts for anchor base in appropriate variant types
+  alleleType: string;
+  sequence: string;
+  mostSevereConsequence: string;
+  hasAnchorBase: boolean;
+  // onClick: () => void;
```

Comment:
Remove comment?

Reply:
This onClick comment and the one you pointed out below is a placeholder for the real click handler coming in a subsequent PR. AlternativeAllele will to respond to clicks.

Reply:
I've eventually added the onClick behaviour (clicking on an allele makes it active).